larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -460,8 +460,6 @@ void ThreeDVertexDistanceFeatureTool::Run(LArMvaHelper::MvaFeatureVector &featur
     if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
         std::cout << "----> Running Algorithm Tool: " << this->GetInstanceName() << ", " << this->GetType() << std::endl;
 
-    LArMvaHelper::MvaFeature vertexDistance;
```

Comment:
Looks like you do want this, actually, as it's a LArMvaHelper::MvaFeature, which is a double wrapped up with a Boolean to describe whether it has been initialized, as per [here](https://github.com/PandoraPFA/LArContent/blob/master/larpandoracontent/LArObjects/LArMvaInterface.h#L82)

At end of function, want to place one of these in the feature vector, rather than some default value.

Reply:
I've reinstated the var